facades/sort_facade.js

```diff
@@ -1,26 +1,43 @@
 var Recipe = require('../models').Recipe;
 var RecipeHelper = require('../helpers/recipe_helper');
 var RecipeSerializer = require('../serializers/recipe_serializer');
-var pry = require('pryjs')
 
 module.exports = class SortFacade {
-  static sortRecipes(search) {
+  static sortRecipes(search, type) {
     return new Promise(function(resolve, reject) {
       checkSearch(search)
       .then(search => {
-        sortSearchedCalories(search)
+        sortSearched(search, type)
         .then(response => {resolve(response)})
         .catch(error => {reject(error)})
       })
       .catch(noSearch => {
-        sortAllCalories()
+        checkSortAllType(type)
         .then(response => {resolve(response)})
         .catch(error => {reject(error)})
       })
     })
   }
 }
 
+function checkSortAllType(type) {
+  if (type === 0){
```

Comment:
It shouldn't happen, but we might want to make an else clause that chooses a sorting in case we happened to forget the type on one. Alternatively, we also might be able to just call the sortAllRecipes function with the type, and hand it down from the controller as a string. Not sure if either would be a better approach. 

Regardless the efforts to reduce duplication here are much appreciated.